java/src/jmri/implementation/configurexml/MatrixSignalMastXml.java

```diff
@@ -30,7 +30,7 @@ public MatrixSignalMastXml() {
     public Element store(Object o) { // from mast p to XML
         MatrixSignalMast p = (MatrixSignalMast) o;
 
-        Element e = new Element("signalmast");
+        Element e = new Element("matrixsignalmast");
```

Comment:
@mattharris Both approaches seem "possible". Would you still prefer to revert the new version and put this into the -2-9-6-mast and -panel? If I manage to do that, are you able to merge?